### PR TITLE
test: Increase pkg/common/utils coverage to more than 90%

### PR DESCRIPTION
**Pull request by liuxianloveqiqi**
#### What type of PR is this?
test

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) Translate the PR title into Chinese.
提高对`pkg/common/utils`的覆盖率至90%以上

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:Increased unit test coverage for `pkg/common/utils` from 88.6% to 91.5%
zh(optional):提高对`pkg/common/utils`的单元测试，覆盖率从88.6%提高到了91.5%

#### (Optional) Which issue(s) this PR fixes:

https://github.com/cloudwego/hertz/issues/257#issuecomment-1722181019

### CODE REVIEW

File: pkg/common/utils/network_test.go [hunk 1]
```diff
@@ -17,9 +17,8 @@
 package utils
 
 import (
-	"testing"
-
 	"github.com/cloudwego/hertz/pkg/common/test/assert"
+	"testing"
```
Comment (Duslia) [inline pkg/common/utils/network_test.go:21]: Please format import

**Comment by Duslia** [comment]
Well done!

File: pkg/common/utils/network_test.go (hunk 1 above)
Comment (liuxianloveqiqi) [reply pkg/common/utils/network_test.go:21]: Thank you, I have noticed the problem

**Comment by welkeyever** [comment]
CI break: Run this before submit: `gofumpt -l -w -extra .`

**Comment by liuxianloveqiqi** [comment]
> CI break: Run this before submit: `gofumpt -l -w -extra .`

Thank you, the first time I contributed to hertz did not understand, now I have formatted the code with gofumpt and commit it now.

**Comment by liuxianloveqiqi** [comment]
@welkeyever Branch merge request

**Comment by liuxianloveqiqi** [comment]
@li-jin-gou apply for merging,please!

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**